packages/headless/src/chat.ts

```diff
@@ -53,11 +61,18 @@ type State = {
    */
   fetching: Record<number, boolean | undefined>;
   /**
-   * Map from room id to error messages
+   * Map from room id to error messages (indicating problem with entire chat)
    */
   errors: Record<number, string | null | undefined>;
```

Comment:
```suggestion
  roomErrors: Record<number, string | null | undefined>;
```

Reply:
Let's make the type more descriptive. 

```suggestion
  type RoomId = number;
  errors: Record<RoomId, string | null | undefined>;
```